charts/newrelic-prometheus-agent/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: newrelic-prometheus-agent
 description: A Helm chart to deploy Prometheus with New Relic Prometheus Configurator.
 type: application
-version: 0.1.1
+version: 0.2.0
 # Prometheus Server Version
 appVersion: "v2.37.1"
```

Comment:
Not sure how the release process would work, but we need for sure to bump this version

Reply:
I meant the version of the configurator 😄

Reply:
oh yea, you could already bump the new version here perhaps?